modules/ROOT/pages/pages-development.adoc

```diff
@@ -127,7 +108,6 @@ If you want to disable it, you need to use the setup tool provided in `workspace
 
 === Constraints
 
-A custom page is displayed in an iframe in the Bonita Portal, so is isolated from changes to the portal.
 When you migrate to a newer version of Bonita, your custom page definition should still be valid.
```

Comment:
When you migrate to a newer version of Bonita, => When you update to a newer version of Bonita,

Reply:
Done